scripts/zones/Garlaige_Citadel_[S]/npcs/Randecque.lua

```diff
@@ -0,0 +1,55 @@
+-----------------------------------
+-- Area: Garlaige Citadel [S]
+--  NPC:Randecque
+-- @pos 61 -6 137 164
+-- Notes: Gives Red Letter required to start "Steamed Rams"
+-----------------------------------
+package.loaded["scripts/zones/Garlaige_Citadel_[S]/TextIDs"] = nil;
+-----------------------------------
+
+require("scripts/globals/settings");
+require("scripts/globals/keyitems");
+require("scripts/zones/Garlaige_Citadel_[S]/TextIDs");
+
+-----------------------------------
+-- onTrade Action
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end;
+
+-----------------------------------
+-- onTrigger Action
+-----------------------------------
+
+function onTrigger(player,npc)
+    if (player:getVar("Used_RED_RECOMMENDATION_LETTER") == 1) then

```

Comment:
Before a certain someone shows up to talk about a certain mission: the mission is not required to trigger any of this. Your mission log change is from the fact you can't pass through a maw to reach these NPC's without triggering that first mission, these NPCs have nothing to do with it and require no change. **Servers allowing players to use the maw before missions exist are the fault of the inconstancy, not the NPC scripts.**